src/lib/loaders/loaders_with_authentication/search.ts

```diff
@@ -0,0 +1,26 @@
+// @ts-check
+import factories from "../api"
+import * as url from "url"
+import { SearchIndex } from "schema/search"
+
+export const searchLoader = (accessToken, opts) => {
+  const gravityAccessTokenLoader = () => Promise.resolve(accessToken)
+  const { gravityLoaderWithAuthenticationFactory } = factories(opts)
+  const gravityLoader = gravityLoaderWithAuthenticationFactory(
+    gravityAccessTokenLoader
+  )
```

Comment:
I think it's better to pass in a preconfigured `gravityLoader` from `src/lib/loaders/loaders_without_authentication/gravity.ts`, because in other cases [than gravity] a service loader will need to do some extra work to fetch a token from e.g. gravity, and we don't want to set an example where people may end up doing the wrong thing. (The wrong thing being to create a new service loader that refetches a token, rather than re-using the already fetched one.)